src/patternfly/components/NotificationDrawer/notification-drawer.scss

```diff
@@ -96,6 +96,7 @@
   --pf-c-notification-drawer__group-toggle-icon--Color: var(--pf-global--Color--200);
   --pf-c-notification-drawer__group-toggle-icon--Transition: .2s ease-in 0s;
   --pf-c-notification-drawer__group--m-expanded__group-toggle-icon--Rotate: 90deg;
+  --pf-c-notification-drawer__header-action--c-dropdown--c-button--MarginLeft: calc(-1 * var(--pf-global--spacer--sm));
```

Comment:
I don't think we need this. It also creates a smaller click target for the dropdown since the button overlaps the right side of the dropdown toggle. The design shows adjacent actions with no spacing.

<img width="914" alt="Screen Shot 2020-08-11 at 9 20 11 AM" src="https://user-images.githubusercontent.com/35148959/89909130-58490c80-dbb4-11ea-8c48-0c1a5ca93011.png">

Though by default, since the `pf-c-notification-drawer__header-action` element is a regular block element, the whitespace between the dropdown and button is preserved, creating extra space.

<img width="222" alt="Screen Shot 2020-08-11 at 9 24 22 AM" src="https://user-images.githubusercontent.com/35148959/89909236-76167180-dbb4-11ea-98f7-be7233b2ec41.png">

We can make that a flex container with `align-items: center;` (so that we don't inadvertently stretch some other element a user may have put in this element), and it looks ok to me. Wdyt?

<img width="196" alt="Screen Shot 2020-08-11 at 9 21 08 AM" src="https://user-images.githubusercontent.com/35148959/89909439-ad851e00-dbb4-11ea-903f-6b1a37ea2ac0.png">

Reply:
Nice! I updated it and i think it looks good too! 
@mcarrano could you check the spacing between the close button and the kebab?

Reply:
@maryshak1996 can you verify?

Reply:
this looks good to me @christiemolloy :)